controller/workitem_blackbox_test.go

```diff
@@ -640,24 +642,36 @@ func createOneRandomUserIdentity(ctx context.Context, db *gorm.DB) *account.Iden
 	}
 	err := identityRepo.Create(ctx, &identity)
 	if err != nil {
-		fmt.Println("should not happen off.")
+		log.Error(ctx, map[string]interface{}{
+			"err": err,
+		}, "failed to create identity")
 		return nil
 	}
 	return &identity
 }
 
 func createOneRandomIteration(ctx context.Context, db *gorm.DB) *iteration.Iteration {
+	spaceTemplate, err := spacetemplate.CreateEmptySpaceTemplate(ctx, db)
+	if err != nil {
+		log.Error(ctx, map[string]interface{}{
+			"err": err,
```

Comment:
Will it make sense to add the spaceTemplate as a property here ? or it'd be a beast